subtrack.Tests/SubscriptionCalculatorTests/GetNextPaymentDateTests.cs

```diff
@@ -39,6 +39,46 @@ public void GetNextPaymentDate_WhenLastPaymentIsLastDayOfMonth_ReturnsPaymentDat
             // Act
             var result = _sut.GetNextPaymentDate(subscription);
 
+            // Assert
+            Assert.Equal(expected, result);
+        }
+        [Fact]
+        public void GetNextPaymentDate_WhenLastPaymentIsLastDayOfFebruary_ReturnsPaymentDateWithLastDayOfNextMonth()
+        {
+            // Arrange
+            var lastPayment = new DateTime(2023, 02, 28);
+            var subscription = new Subscription { LastPayment = lastPayment, FirstPaymentDay = lastPayment.Day };
+            var expected = new DateTime(2023, 03, 31);
```

Comment:
If the firstpaymentday is 28th, it should always be 28th if every month has 28 days